packages/stable/src/api/vision/types.gen.ts

```diff
@@ -3,30 +3,47 @@
 // Instead update the code generation logic or the OpenAPI document.
 
 /**
+ * primitives.references.AssetRef
  * A reference to an asset. Either the internal ID or the external ID must be provided (exactly one).
  */
 export type AnnotationsAssetRef = {
+  /**
+   * The internal ID of the referenced resource
+   * @default null
+   */
   id?: number;
+  /**
+   * The external ID of the referenced resource
+   * @maxLength 256
+   * @default null
+   */
   externalId?: string;
 };
 /**
+ * primitives.attributes.Boolean
```

Comment:
do you know if we can remove these `primitives.attributes` annotations? they seem useless